api/openapi.yaml

```diff
@@ -245,7 +275,7 @@ components:
           type: string
           minLength: 1
           maxLength: 63
-          pattern: "^[a-z0-9]+(?:[_-][a-z0-9]+)*$"
+          pattern: "^[a-z0-9]+(?:_[a-z0-9]+)*$"
```

Comment:
What's the purpose of this change?

Reply:
Not allowing dash is both a devex decision to make it easier to copy slugs, also compatibility decision as it goes into table name.